src/Request/CustomFields/CustomField.php

```diff
@@ -8,16 +11,42 @@
 
 class CustomField extends Model
 {
-    const TEXT = 'text';
-    const TEXT_AREA = "textarea";
-    const DATE = "date";
-    const CHECKBOX = "checkbox";
-    const RADIO = "radio";
-    const SELECT = "select";
+    /**
+     * @var string
+     */
+    public const TEXT = 'text';
+
+    /**
```

Comment:
u tich constant bych asi zrušil typehint typu co? Nevíš jak odstranit tu mezeru?

Reply:
To pridava rector ale imho bych tohle pravidlo taky odstranil. Jakou mezeru myslis?

Reply:
Jiný rector / ecs dává extra řádek mezi constantama - myslím že to dělá zbytečně.